shell/browser/native_window_views.cc

```diff
@@ -1062,6 +1066,18 @@ void NativeWindowViews::SetIgnoreMouseEvents(bool ignore, bool forward) {
           .source_bitmap = x11::Pixmap::None,
       });
     }
+  } else {
+    // note: this assumes we are in wayland, but Ozone supports X11 too
+    // is there a way to confirm we are indeed on the wayland ozone platform?
+    ui::WaylandToplevelWindow* waylandToplevelWindow =
+        static_cast<ui::WaylandToplevelWindow*>(
+            views::DesktopWindowTreeHostPlatform::GetHostForWidget(
+                GetAcceleratedWidget())
+                ->platform_window());
+    gfx::Rect rect =
+        ignore ? gfx::Rect(0, 0) : waylandToplevelWindow->GetBounds();
+    waylandToplevelWindow->root_surface()->SetInputRegion(rect);
+    waylandToplevelWindow->root_surface()->Commit();
```

Comment:
I'm not familiar enough neither with Electron nor Chromium but I get the feeling that there *might* be a better way to schedule a surface commit without explicitly calling it here. Unfortunately I don't know what it is but most likely the Electron developers might be able to suggest a better approach when reviewing the changes.